testplan/testing/multitest/entries/base.py

```diff
@@ -345,21 +346,31 @@ def assert_valid_series_options(self, series_options, graph_data):
 class Attachment(BaseEntry):
     """Entry representing a file attached to the report."""
 
-    def __init__(self, filepath, description, dst_path=None):
+    def __init__(
+        self, filepath, description, dst_path=None, scratch_path=None
+    ):
+
         self.source_path = filepath
         self.orig_filename = os.path.basename(filepath)
         self.hash = hash_file(filepath)
         self.filesize = os.path.getsize(filepath)
+        basename, ext = os.path.splitext(self.orig_filename)
+
         if dst_path:
             self.dst_path = dst_path
         else:
-            basename, ext = os.path.splitext(self.orig_filename)
-            self.dst_path = "{basename}-{hash}-{filesize}{ext}".format(
-                basename=basename,
-                hash=self.hash,
-                filesize=self.filesize,
-                ext=ext,
-            )
+            # below logic to avoid file name collision
+            self.dst_path = f"{basename}-{self.hash}-{self.filesize}{ext}"
+
```

Comment:
can be:
self.dst_path = dest_path or f"{basename}-{self.hash}-{self.filesize}{ext}"